### ISSUE TITLE
:tractor: Migrate from SQLite to Postgres

### ISSUE THREAD

**Issue by ryancheley** [ryancheley/acronym-slackbot#138]
Steps for migration:

- [x] dump the data from the SQLite database with command `python manage.py dumpdata --exclude=admin_interface -o acronyms.json`
- [x] Update connection string to Postgres dB (via a deployment)
- [x] Update the `.env` on the server with new dB connection
- [x] run migrations `python manage.py migrate`
- [x] load the data from acronyms.json with command `python manage.py loaddata acronyms.json`
- [x] archive old SQLite database into S3

**Comment by ryancheley** [comment]
I've run into an issue with the load command locally. The following error is generated:

```
Traceback (most recent call last):
  File "/Users/ryan/Documents/github/acronym-slackbot/venv/lib/python3.10/site-packages/django/db/models/options.py", line 672, in get_field
    return self.fields_map[field_name]
KeyError: 'language_chooser_control'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ryan/Documents/github/acronym-slackbot/venv/lib/python3.10/site-packages/django/core/serializers/json.py", line 70, in Deserializer
    yield from PythonDeserializer(objects, **options)
  File "/Users/ryan/Documents/github/acronym-slackbot/venv/lib/python3.10/site-packages/django/core/serializers/python.py", line 131, in Deserializer
    field = Model._meta.get_field(field_name)
  File "/Users/ryan/Documents/github/acronym-slackbot/venv/lib/python3.10/site-packages/django/db/models/options.py", line 674, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: Theme has no field named 'language_chooser_control'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/ryan/Documents/github/acronym-slackbot/manage.py", line 22, in <module>
    main()
  File "/Users/ryan/Documents/github/acronym-slackbot/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/Users/ryan/Documents/github/acronym-slackbot/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
    utility.execute()
  File "/Users/ryan/Documents/github/acronym-slackbot/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 440, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/ryan/Documents/github/acronym-slackbot/venv/lib/python3.10/site-packages/django/core/management/base.py", line 414, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/ryan/Documents/github/acronym-slackbot/venv/lib/python3.10/site-packages/django/core/management/base.py", line 460, in execute
    output = self.handle(*args, **options)
  File "/Users/ryan/Documents/github/acronym-slackbot/venv/lib/python3.10/site-packages/django/core/management/commands/loaddata.py", line 102, in handle
    self.loaddata(fixture_labels)
  File "/Users/ryan/Documents/github/acronym-slackbot/venv/lib/python3.10/site-packages/django/core/management/commands/loaddata.py", line 163, in loaddata
    self.load_label(fixture_label)
  File "/Users/ryan/Documents/github/acronym-slackbot/venv/lib/python3.10/site-packages/django/core/management/commands/loaddata.py", line 251, in load_label
    for obj in objects:
  File "/Users/ryan/Documents/github/acronym-slackbot/venv/lib/python3.10/site-packages/django/core/serializers/json.py", line 74, in Deserializer
    raise DeserializationError() from exc
django.core.serializers.base.DeserializationError: Problem installing fixture '/Users/ryan/Documents/github/acronym-slackbot/acronyms.json': 
```

The error does not happen when I exclude the model `admin_interface`. I believe this means that a field was added to that app and then removed. I'm not sure why the migrations wouldn't be picking it up though so 🤔

**Comment by ryancheley** [comment]
excluding the `admin_interface` app resolved the above issue

**Comment by ryancheley** [comment]
closed with #142
